File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        index = 1
        for i in range(1, len(nums)):
            if(nums[i] != nums[i-1]):
                nums[index] = nums[i]
                index += 1
        return index
            

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int ans = 0, before = nums[0];
        for(int i=1; i<nums.length; i++){
            if(before != nums[i]){
                nums[++ans] = nums[i];
                before = nums[i];
            }
        }
        return ans+1;
    }
}